JS-Algorithms/challenges.js

```diff
@@ -13,7 +13,20 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 ***** */
 
 const readableTime = (seconds) => {
-  // YOUR CODE HERE...
+  if (seconds < 0) {
+    throw new Error(`Invalid Input Negative Number: ${seconds}`);
+  }
+  let resSeconds = seconds;
+  const hours = Math.round(resSeconds / 3600, 0);
+  resSeconds %= 3600;
+  const minutes = Math.floor(resSeconds / 60, 0);
+  resSeconds %= 60;
+
+  const hh = hours > 9 ? hours.toString() : `0${hours}`;
+  const mm = minutes > 9 ? minutes.toString() : `0${minutes}`;
+  const ss = resSeconds > 9 ? resSeconds.toString() : `0${resSeconds}`;
```

Comment:
you don't need to parse the numbers to string since on line 29 you're using a template string.

Reply:
yes, you're right. They were unnecessary. I thought I needed convert every value to string. Thanks for the feedback